ObsoleteFiles.inc

```diff
@@ -51,6 +51,10 @@
 #   xargs -n1 | sort | uniq -d;
 # done
 
+# 20240411: Removal of ld-elf*-c18n.so.1
```

Comment:
To be clear, here you want to remove not the source files, but the things that get installed.  So in this case you want `libexec/ld-elf-c18n.so.1`, etc.